packages/flutter/lib/src/widgets/scroll_delegate.dart

```diff
@@ -413,6 +413,60 @@ class SliverChildBuilderDelegate extends SliverChildDelegate {
   /// none of the children will ever try to keep themselves alive.
   ///
   /// Defaults to true.
+  ///
+  /// {@tool dartpad}
+  /// This sample demonstrates how to use the [AutomaticKeepAlive] widget in
+  /// combination with the [AutomaticKeepAliveClientMixin] to selectively preserve
+  /// the state of individual items in a scrollable list.
+  ///
+  /// Normally, widgets in a lazily built list like [ListView.builder] are
+  /// disposed of when they leave the visible area to save resources. This means
+  /// that any state inside a [StatefulWidget] would be lost unless explicitly
```

Comment:
Is it okay to push code examples inside documentation templates?

Reply:
I think so!